targets/unity-v2/Testing/Editor/PlayFabPackager.cs

```diff
@@ -3,9 +3,49 @@
 using System.Reflection;
 using UnityEngine;
 using UnityEditor;
+using UnityEditor.iOS;
+using UnityEditor.iOS.Xcode;
+using UnityEditor.Build;
+using UnityEditor.Build.Reporting;
+
+using BuildPipeline = UnityEditor.BuildPipeline;
 
 namespace PlayFab.Internal
 {
+    class MyCustomBuildProcessor : IPostprocessBuildWithReport
```

Comment:
Should we change it to something like PlayFabTestBuildProcessor?

Reply:
Yes.  I'll rename it to something other than the generated name.